### PR TITLE
Docs: Improve "of" grammar

### PR DESCRIPTION
**Pull request by johnlow95**

### CODE REVIEW

File: doc/translation_process.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ token =
 username = USERNAME
 ```
 
-The Transifex Bitcoin project config file is included as part of the repo. It can be found at `.tx/config`, however you shouldn’t need change anything.
+The Transifex Bitcoin project config file is included as part of the repo. It can be found at `.tx/config`, however you shouldn’t need  to change anything.
```
Comment (Empact) [inline doc/translation_process.md]: You have an extra space here between "need" and "to"

Comment (johnlow95) [reply doc/translation_process.md]: alright, will be changing it now

**Comment by Empact** [comment]
ACK 1b3d63f - you can title this "Docs:" https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#contributor-workflow

**Comment by ken2812221** [comment]
Please squash

 https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by johnlow95** [comment]
done

**Comment by johnlow95** [comment]
the squash mean? because i done it by using github webpage

**Comment by Empact** [comment]
@johnlow95 You could close this and open a new PR with a single edit, rather than 2.

**Comment by johnlow95** [comment]
oh...thanks for the guiding, will do it now

**Comment by Empact** [comment]
Please close this one too.

**Pull request closed without merging**